x/cdp/keeper/keeper.go

```diff
@@ -110,3 +110,23 @@ func (k Keeper) IterateCdpsByCollateralRatio(ctx sdk.Context, collateralType str
 
 	}
 }
+
+// SetSavingsRateDistributed sets the SavingsRateDistributed in the store
+func (k Keeper) SetSavingsRateDistributed(ctx sdk.Context, totalDistributed sdk.Int) {
+	store := prefix.NewStore(ctx.KVStore(k.key), types.SavingsRateDistributedKey)
+	bz := k.cdc.MustMarshalBinaryLengthPrefixed(totalDistributed)
+	store.Set(types.SavingsRateDistributedKey, bz)
+}
+
+// GetSavingsRateDistributed gets the SavingsRateDistributed from the store
+func (k Keeper) GetSavingsRateDistributed(ctx sdk.Context) sdk.Int {
+	var savingsRateDistributed sdk.Int
+	store := prefix.NewStore(ctx.KVStore(k.key), types.SavingsRateDistributedKey)
+	bz := store.Get(types.SavingsRateDistributedKey)
+	if bz == nil {
+		return savingsRateDistributed
```

Comment:
this will return an uninitialized `Int`, better use `sdk.ZeroInt` if you are not returning a boolean or an error

Reply:
https://github.com/Kava-Labs/kava/pull/644/commits/2ffcdfa8feb3fb5916bc0eb63d91669d1f65f6ba